File: antisocial-bot/manual_mission_tweets.py


import tweepy
import time
import sys
import os

# Twitter API credentials - Set via environment variables
# Get credentials from: https://developer.twitter.com/en/portal/dashboard
API_KEY = os.getenv('TWITTER_API_KEY')
API_SECRET = os.getenv('TWITTER_API_SECRET')
ACCESS_TOKEN = os.getenv('TWITTER_ACCESS_TOKEN')
ACCESS_SECRET = os.getenv('TWITTER_ACCESS_SECRET')

# The 3 Specific Mission Tweets
TWEETS = [
    "Direction: We are building the identity layer for competitive gaming. No aggressive monetization, no promised riches. Just a loop: Play -> Verify -> Earn. V1 is live. #ggloop",
    "Founder update: Personal site is live. Theme is locked. Now focusing 100% on the loop mechanics. Transparency is the only marketing strategy we need.",
    "Trust Score V1 is live. It penalizes smurfs and rewards consistency. If you verify your Riot ID and play on Desktop, you’re already earning. Check your dashboard."
]

def post_mission_tweets():
    print("🐦 Initializing Mission 2 Tweet Sequence...")
    
    try:
        client = tweepy.Client(
            consumer_key=API_KEY,
            consumer_secret=API_SECRET,
            access_token=ACCESS_TOKEN,
            access_token_secret=ACCESS_SECRET
        )
        
        # Verify credentials by getting me
        me = client.get_me()
        print(f"✅ Authenticated as: @{me.data.username}")
        
    except Exception as e:
        print(f"❌ Authentication Failed: {e}")
        sys.exit(1)

    for i, text in enumerate(TWEETS, 1):
        print(f"\n[{i}/3] Posting: {text[:50]}...")
        try:
            response = client.create_tweet(text=text)
            print(f"   ✅ Posted! ID: {response.data['id']}")
            
            if i < len(TWEETS):
                print("   ⏳ Waiting 60 seconds...")
                time.sleep(60)
                
        except Exception as e:
            print(f"   ❌ Failed to post tweet {i}: {e}")
            # Continue to next tweet even if one fails? Maybe stop?
            # Let's stop to be safe.
            sys.exit(1)

    print("\n✅ Mission 2 Complete: All tweets posted.")

if __name__ == "__main__":
    post_mission_tweets()

File: antisocial-bot/manual_mission_tweets.ts

import { TwitterApi } from 'twitter-api-v2';

// 🛑 STOP: You must add your valid Twitter API Credentials below
// The credentials found in temp_hunter_2/EMPIRE-MEMORY.md were REVOKED (401 Unauthorized).
// Please generate new keys at developer.twitter.com or copy them from Railway Dashboard.

const appKey = "YOUR_CONSUMER_KEY";
const appSecret = "YOUR_CONSUMER_SECRET";
const accessToken = "YOUR_ACCESS_TOKEN";
const accessSecret = "YOUR_ACCESS_TOKEN_SECRET";

const client = new TwitterApi({
    appKey,
    appSecret,
    accessToken,
    accessSecret,
});

const TWEETS = [
    "Direction: We are building the identity layer for competitive gaming. No aggressive monetization, no promised riches. Just a loop: Play -> Verify -> Earn. V1 is live. #ggloop",
    "Founder update: Personal site is live. Theme is locked. Now focusing 100% on the loop mechanics. Transparency is the only marketing strategy we need.",
    "Trust Score V1 is live. It penalizes smurfs and rewards consistency. If you verify your Riot ID and play on Desktop, you’re already earning. Check your dashboard."
];

async function main() {
    console.log("🐦 Initializing Mission 2 Tweet Sequence (Node.js)...");

    if (appKey === "YOUR_CONSUMER_KEY") {
        console.error("❌ ERROR: You must update this script with valid Twitter Credentials before running.");
        process.exit(1);
    }

    try {
        const me = await client.v2.me();
        console.log(`✅ Authenticated as: @${me.data.username}`);
    } catch (e) {
        console.error(`❌ Authentication Failed:`, e);
        process.exit(1);
    }

    for (let i = 0; i < TWEETS.length; i++) {
        const text = TWEETS[i];
        console.log(`\n[${i + 1}/3] Posting: ${text.substring(0, 50)}...`);

        try {
            const response = await client.v2.tweet(text);
            console.log(`   ✅ Posted! ID: ${response.data.id}`);

            if (i < TWEETS.length - 1) {
                console.log("   ⏳ Waiting 60 seconds...");
                await new Promise(resolve => setTimeout(resolve, 60000));
            }
        } catch (e) {
            console.error(`   ❌ Failed to post tweet ${i + 1}:`, e);
            process.exit(1);
        }
    }

    console.log("\n✅ Mission 2 Complete: All tweets posted.");
}

main();
